php/php.editor/test/unit/data/testfiles/structure/php53/multiple_namespaces_with_elements_from_default.php

```diff
@@ -0,0 +1,10 @@
+
+<?php
+class Connection {}
+namespace MyProject;
+class Connection {}
+function connect() {}
+namespace AnotherProject;
+class Connection {}
+function connect() {}
```

Comment:
Note: NB doesn't handle this as an error but it is fatal error. We cannot write classes before the Namespace declaration.

Reply:
Sorry, my bad. I've removed the test.